py3status/modules/deadbeef.py

```diff
@@ -58,6 +69,13 @@ def post_config_hook(self):
         self.color_playing = self.py3.COLOR_PLAYING or self.py3.COLOR_GOOD
         self.color_stopped = self.py3.COLOR_STOPPED or self.py3.COLOR_BAD
 
+        # mix format+hidden placeholders with format+control separators...
+        # then we merge strings together to run deadbeef command only once
+        self.placeholders = self.py3.get_placeholders_list(self.format)
+        out = FORMAT_SEPARATOR.join(['%{}%'.format(x) for x in self.placeholders])
+        CONTROL = [CONTROL_SEPARATOR + s for s in (CONTROL_PARAMETERS)]
+        self.fmt = ''.join(out) + ''.join(CONTROL)
```

Comment:
As I said before we do not need all this complication, there is nothing special about your `CONTROL_PARAMETERS` so do not treat them as such they are just 2 extra placeholders.

`placeholders = list(set(self.py3.get_placeholders_list(self.format)) | set(CONTROL_PARAMETERS))`